tapfreighter/wallet.go

```diff
@@ -784,6 +785,8 @@ func (f *AssetWallet) fundPacketWithInputs(ctx context.Context,
 				ScriptKey: asset.NUMSScriptKey,
 			}
 
+			// TODO(roasbeef): inherit version from which input?
```

Comment:
How about the max version of all inputs? E.g. if one is V1 but the rest isn't, we'll probably not want to downgrade one but instead upgrade the others?